config/config.yml

```diff
@@ -3,3 +3,5 @@ rest:
   host: sandbox.dspace.org
   port: 443
   nameSpace: /server
+item:
+  showAltmetricBadge: true
```

Comment:
This should be removed as it shouldn't be necessary.  All default config values are only in `app/config/default-app-config.ts`.  (That's why this file has very few configurations in it)